ipn/localapi/localapi.go

```diff
@@ -179,22 +179,28 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// validLocalHost allows either localhost or loopback IP hosts on platforms
+// that use token security.
+var validLocalHost = runtime.GOOS == "darwin" || runtime.GOOS == "android"
```

Comment:
@mihaip, do we need `"ios"` too?

Actually, I don't think any of those three platforms use the ipnserver code, so is this code even exercised? This whole const seems misleading then.

Reply:
Yeah, we'd need `ios`. Both it and macOS/`darwin` end up making requests to `localhost` (https://github.com/tailscale/corp/blob/24f5a2d230a0d27b9665e0d5c33eeabbcdd11dbf/xcode/Shared/LocalAPIClient.swift#L320), they should be hitting this code path.

Reply:
Yeah, okay, I see the `localapi.NewHandler` call site on the macOS/iOS side now. I was thinking this check was only for ipnserver but that's wrong... it's just checking the http.Request.Host which'll still be checked for macOS/iOS.

Reply:
added ios